test.test_simulation.TestNoisyLTFArray.test_init_normal_empirical Is Not Deterministic

The test case `test_init_normal_empirical` in test module `test.test_simulation.TestNoisyLTFArray` is not deterministic and sometimes generates the following output:
```
======================================================================
FAIL: test_init_normal_empirical (test.test_simulation.TestNoisyLTFArray)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/chris/arbeit/pypuf/test/test_simulation.py", line 756, in test_init_normal_empirical
    self.assertTrue(abs(tools.approx_dist(nla, nla, 10000) - intra_dist) < .02)
AssertionError: False is not true
```
I this phänomenon occured under python 3.4.5 and packages:
```
astroid (1.5.3)
isort (4.2.15)
lazy-object-proxy (1.3.1)
mccabe (0.6.1)
numpy (1.13.3)
pep8 (1.7.1)
pip (9.0.1)
polymath (0.1.17)
pylint (1.7.5)
pypuf-helper (0.1.4)
scipy (0.19.1)
setuptools (27.2.0)
six (1.11.0)
wheel (0.29.0)
wrapt (1.10.11)
```

--- a/pypuf/simulation/arbiter_based/ltfarray.py
+++ b/pypuf/simulation/arbiter_based/ltfarray.py
@@ -732,7 +732,7 @@ class NoisyLTFArray(LTFArray):
 
         # binary search in [0, max_sigma_noise]
         instance.sigma_noise = (max_sigma_noise + min_sigma_noise) / 2
-        estimation_distance = tools.approx_dist(instance, instance, 10000)
+        estimation_distance = tools.approx_dist(instance, instance, 10000, random_instance)
         while abs(intra_dist - estimation_distance) > approx_threshold:
 
             # update interval bounds
@@ -743,7 +743,7 @@ class NoisyLTFArray(LTFArray):
 
             # update instance and estimated distance
             instance.sigma_noise = (max_sigma_noise + min_sigma_noise) / 2
-            estimation_distance = tools.approx_dist(instance, instance, 10000)
+            estimation_distance = tools.approx_dist(instance, instance, 10000, random_instance)
 
         return instance
 

